app/javascript/react/components/RestaurantTile.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const RestaurantTile = (props) => {
+
+  return (
+    <div className="tile cell small-12 medium-4 large-4">
+      <img className="tile-image" src={props.restaurant.picture_url}/>
+      <div className="tile-text">{props.restaurant.name}</div>
+    </div>
+  )
+}
+export default RestaurantTile
```

Comment:
styling edit: remove line 4 and add a line below 11

Reply:
Fixed the styling edit in the most recent PR (#5)